.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - feature/pplus-105/implement-cli-commands
+  workflow_dispatch:
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.3',
+            '7.4'
```

Comment:
should be 8.0 now
where did this template come from?

Reply:
Added 8.0 
Template https://github.com/spryker/architecture-sniffer/blob/master/.github/workflows/ci.yml#L17